less/home-page.less

```diff
@@ -57,5 +57,78 @@
       width: 30%;
       margin-bottom: 30px;
     }
+    
+      .btn{
+        font-size: 2.2rem;
+      border:1px solid #17A2B8;
+      border-radius:8px;
+      text-align: center;
+      padding:15px;
+      .bottombuttons(@button-bg,@white);
+      a{
+        text-decoration: none;
+        color:white;
+      }
+    }
+  }
+  
+
+
+  
+           
+}
+
+@media @mobile{
+  nav{
+    display:flex;
+    flex-direction: column;
+    padding:0;
+    h1{
+      margin:0 auto 10px;
+      text-align: center;
+    }
+    div{
+      width:100%;
+      text-align: center;
+    }
+    a{
+      margin: 5px;
+    }
   }
-}// home
\ No newline at end of file
+.content-section{
+  display:flex;
+  flex-direction: column;
+  align-items: center;
+  p{
+    // text-align:center;
+   
+  }
+}
+  header{
+    h1{
+      font-size: 2.5rem;
+      width:90%;
+    }
+  }
+    .inverse-content{
+      display:flex;
+      flex-direction: column-reverse;
+      align-content: center;
+    }
+ .home .content-pick{
+    width:100%;
+  display:flex;
+    flex-direction: column;
+    align-items: center;
+    // border:3px solid blue;
+    .destination{
+      width:100%;
+      display: flex;
+      flex-direction: column;
+      // border:3px solid red;
+        }
+  }
+
+
+
+}
```

Comment:
nice less! If you havent already , look into a VSCode Extension called `Prettier` - it's a code formatter to help keep code organized & formatted! (: